docs/reference/cluster/nodes-stats.asciidoc

```diff
@@ -338,7 +338,7 @@ Supported metrics are:
 * `flush`
 * `get`
 * `indexing`
-* `merge`
+* `merges`
```

Comment:
Did you try this on master and 6.4? I get errors when using "merges":
```
				"reason": "request [/_nodes/stats/indices/merges] contains unrecognized index metric: [merges] -> did you mean [merge]?"
```
But "merge" seems to work on master and 6.4? Maybe this is a change that is needed for lower versions though?

Reply:
Great catch and thanks for reviewing @cbuescher !

So I tried e.g. with 6.5 and hitting `merge` works indeed as suggested in the docs:

`_nodes/stats/indices/merge`

_however_ it returns `merges`:

```
...
     "indices": {
        "merges": {
          "current": 0,
          "current_docs": 0,
          "current_size_in_bytes": 0,
          "total": 136,
          "total_time_in_millis": 1069825,
          "total_docs": 799247,
          "total_size_in_bytes": 31356095062,
          "total_stopped_time_in_millis": 0,
          "total_throttled_time_in_millis": 345938,
          "total_auto_throttle_in_bytes": 278802617
        }
...
```

I am thinking that this discrepancy is a bit trappy, as apart from directly pointing at `_node/stats/indices/<metric>` also apps may just retrieve that entire output of `_node/stats/indices` and filter based on the output, or quite simply with filter_stats one needs to use `merges` instead of `merge`:

```
_nodes/stats?filter_path=**.merges
```

Any thoughts if it makes sense to change either the endpoint or the name of the key/metric in the response?

Reply:
> Any thoughts if it makes sense to change either the endpoint or the name of the key/metric in the response?

I'm unsure about this. If we change either the endpoint of the response field, we are probably going to break some stuff. Both seem to be in place this way for quite some time. I'm not sure if the inconsistency justifies a breaking change. But I'm interested in hearing other peoples opinions. Maybe if would make sense to open a new issue about a change like that instead and close this PR?

Reply:
@cbuescher I agree, it sounds risky and definitely doesn't deserve getting addressed in this PR. I'll close it out and open a new issue instead.